style.css

```diff
@@ -0,0 +1,178 @@
+body {
+  background-color: #eceef1;
+}
+
+.content {
+  display: grid;
+  width: 100%;
+  grid-auto-rows: minmax(30px, auto);
+}
+
+.content .grid {
+  width: 390px;
+  margin: 0 auto;
+}
+
+.content .one {
+  margin-right: 0;
+  height: 75px;
+}
+
+.content .two {
+  height: 50px;
+  text-justify: center;
+  text-align: center;
+}
+
+.content .three {
+  height: 45px;
+  text-align: center;
+}
+
+.content .four {
+  height: 670px;
+  border: solid;
+  border-color: lightgray;
+  border-width: 1px;
+  background-color: white;
+  border-bottom: none;
+  padding: 35px 30px;
+}
+
+.four p {
+  text-align: center;
+}
+
+.main_content p {
+  font-size: 30px;
+}
+
+.para {
+  margin-bottom: -12px;
+}
+
+.four input {
+  width: 100%;
+  height: 33px;
+  border-radius: 5px;
+  border: solid;
+  border-color: lightgray;
+}
+
+#email {
+  margin-top: 15px;
+  margin-bottom: 30px;
+}
+
+#phone {
+  margin-top: 15px;
+}
+
+.phone_detail {
+  text-align: left !important;
+  font-size: 13px;
+  margin-top: 2px;
+  margin-bottom: 30px;
+}
+
+#pass {
+  margin-top: 15px;
+  margin-bottom: 30px;
+}
+
+.four button {
+  display: block;
+  margin: 20px auto;
+  width: 100%;
+  height: 50px;
+  border-radius: 5px;
+  border: none;
+  box-shadow: 0 1px lightgray;
+  background-color: #0077c5;
+  color: white;
+}
+
+.main_content {
+  margin: 0 auto;
+  text-align: center;
+}
+
+.content .five {
+  border: solid;
+  border-color: lightgray;
+  border-width: 1px;
+  border-top: none;
+  padding: 0 30px;
+  font-size: 14px;
+  color: #8d90a1;
+}
+
+a {
+  text-decoration: none;
+  color: #0077c5;
+}
+
+.content .six {
+  height: 77px;
+  width: 730px;
+  text-align: center;
+  font-size: 13px;
+}
+
+.six p,
+.six a {
+  color: #505050 !important;
+  line-height: 18px;
+}
+
+.six a:hover {
+  text-decoration: underline !important;
+}
+
+.one .one_flex {
+  display: flex;
+}
+
+.sign_text {
+  flex-grow: 3;
+  text-align: right;
+  padding-right: 15px;
+  line-height: 23px;
+}
+
+.sign_in {
+  border: solid;
+  border-radius: 10px;
+  background-color: #0077c5;
+  align-self: center;
+  flex-grow: 23;
+  justify-self: center;
+  margin-top: 10px;
+  width: 100px;
+  margin-right: 15px;
+}
+
+button img {
+  display: inline-block;
+  height: 15px;
+  width: auto;
+  margin-left: 10px;
+  position: relative;
+  top: 2px;
+  margin-right: 5px;
+}
+
+.sign_in img {
+  display: inline-block;
+  height: 15px;
+  width: auto;
+  margin-left: 10px;
+  position: relative;
+  top: 2px;
+}
```

Comment:
- [ ] Here, make sure you have no repetition of the same rules. Both selectors have almost the same styling. You should consider merging both stylings into one general class and add the class name to those elements. It is okay to have multiple classes for an element but inefficient to repeat the same CSS rules.